jupyterdrive/gdrive/drive-contents.js

```diff
@@ -116,6 +116,18 @@ define(function(require) {
             model['content'] = notebook_model.notebook_from_file_contents(contents);
             model['writable'] = metadata['editable'];
             return model;
+        }).catch(function(data){   
+            var reason ='Unknown Error.';
+            if( data.xhr.status === 404){
+                reason = 'We cannot access requested resource\n'+
+                         'This can happend if the resource was not created with jupyter drive.\n'+
+                         'Please re-upload the resource through jupyter-drive and re-try.';
```

Comment:
This isn't clear own what the user should do.  How about "Please re-upload the resource by dragging a copy of the file onto the Jupyter file manager".  Dragging a file into tree view doesn't actually work yet, but we can fix it.